mock/src/styles/main.css

```diff
@@ -0,0 +1,83 @@
+/*
+  This is a reasonable starting point for the CSS in Mock.
+  Positioning in CSS can sometimes be annoying to manage, so we've opted to 
+  give you this starter CSS. Feel free to modify it as you wish, though.
+*/
+
+.repl {
+  position: relative;
+  min-height: 95vh;
+}
+
+/* .repl-history {
```

Comment:
For style and readability, make sure to delete commented out code. Though I know these small things are easy to miss :)